crypto/object-hasher/test/index.ts

```diff
@@ -1,13 +1,25 @@
 import { hashObject, hashObjectWithoutSorting } from '@pnpm/crypto.object-hasher'
 
 describe('hashObject', () => {
+  const hash = hashObject
```

Comment:
This is just to make copy/pasting tests easier. I made mistakes while writing the tests until I realized I was testing the wrong function.